app/Services/CampaignFollowService.php

```diff
@@ -61,6 +63,8 @@ public function add(Campaign $campaign, User $user): bool
         $follow = new CampaignFollower();
         $follow->campaign_id = $campaign->id;
         $follow->user_id = $user->id;
+        $campaign->follower++;
```

Comment:
Same thing as for deleting, we're better off adding this logic in `app\Observers\CampaignFollowerObserver::created` method. Check out https://laravel.com/docs/9.x/eloquent#observers on how we can use some Laravel goodness in this case :)